packages/app-provider/package.json

```diff
@@ -6,8 +6,10 @@
   "main": "dist/index.js",
   "module": "dist/index.js",
   "scripts": {
-    "build": "../../bin/build-package.sh",
-    "watch": "../../bin/watch-package.sh",
+    "build:clean": "rm -rf dist",
```

Comment:
@ej9x I think you should use `rimraf` instead of `rm`. `rimraf` is cross-platform solution.

Reply:
fixed